Mage/src/main/java/mage/game/permanent/token/BlueBlackZombieRogueToken.java

```diff
@@ -0,0 +1,26 @@
+package mage.game.permanent.token;
+
+import mage.MageInt;
+import mage.constants.CardType;
+import mage.constants.SubType;
+
+public final class BlueBlackZombieRogueToken extends TokenImpl {
```

Comment:
The zombie token is already there from a commit earlier.

`new ZombieRogueToken()`